pyproject.toml

```diff
@@ -67,11 +67,12 @@ version_scheme = "no-guess-dev"
 
 [tool.pytest.ini_options]
 addopts = """
+    --import-mode importlib
     --cov validate_pyproject
     --cov-report term-missing
     --doctest-modules
-    --verbose
     --strict-markers
+    --verbose
 """
 norecursedirs = ["dist", "build", ".tox"]
```

Comment:
No, I don’t believe they are redundant, because you can manually pass in testpaths. That setting is just a default, while this one makes sure that things that might contain tests (like virtual environments) never get picked up.